plugin.rb

```diff
@@ -472,6 +472,10 @@ def can_accept_answer?(topic, post)
       return true if is_staff?
       return true if current_user.trust_level >= SiteSetting.accept_all_solutions_trust_level
 
+      if Guardian.new.respond_to? :can_perform_action_available_to_group_moderators?
```

Comment:
```suggestion
      if respond_to? :can_perform_action_available_to_group_moderators?
```
Am I missing something or would this work? ^

Reply:
👍